From: Kent Overstreet <kent.overstreet@linux.dev>
Subject: fixup! fix missing vmalloc.h includes
Date: Tue, 26 Mar 2024 20:21:52 -0400

The previous patch added string.h to arch/alpha/lib/memcpy.c to fix a
warning about a missing forward declaration - but this breaks with -
presumably - some fortify source shenanigans that redefine memcpy as a
macro.

Link: https://lkml.kernel.org/r/20240327002152.3339937-1-kent.overstreet@linux.dev
Signed-off-by: Kent Overstreet <kent.overstreet@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/alpha/lib/memcpy.c |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/alpha/lib/memcpy.c~fix-missing-vmalloch-includes-fix
+++ a/arch/alpha/lib/memcpy.c
@@ -18,7 +18,6 @@
 
 #include <linux/types.h>
 #include <linux/export.h>
-#include <linux/string.h>
 
 /*
  * This should be done in one go with ldq_u*2/mask/stq_u. Do it
_
